src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -1145,35 +1205,29 @@ class InventoryRowModel extends DefaultTableModel {
 		private static final long serialVersionUID = 1L;
 
 		public InventoryRowModel(boolean add) throws OHServiceException {
+			inventoryRowList.clear();
 			inventoryRowList = loadNewInventoryTable(null, inventory, add);
 			if (!inventoryRowList.isEmpty()) {
 				for (MedicalInventoryRow invRow : inventoryRowList) {
 					addMedInRowInInventorySearchList(invRow);
 				}
-			} else {
-				MessageDialog.info(null, "angal.inventory.youhavealreadyaddedallproduct.msg");
 			}
 			if (areAllMedicalsInInventory()) {
-				allRadio.setSelected(true);
 				specificRadio.setSelected(false);
 				codeTextField.setEnabled(false);
 			} else {
 				specificRadio.setSelected(true);
 				codeTextField.setEnabled(true);
-				allRadio.setSelected(false);
 			}
```

Comment:
By effect of my previous comment, this block can be removed in each constructor.